docs/release/release_0_3_0.md

```diff
@@ -0,0 +1,278 @@
+explains how to embed a matplotlib plot within napari.
+
+### Multiscale image handling
+
+napari is now much better at handling large datasets. Viewing a large dataset
+will no longer trigger automatic — but very slow, and often unnecessary —
+generation of an image pyramid. Instead, we direct users to our [tutorial on
+how to generate your own pyramid](https://scikit-image.org/docs/dev/auto_examples/transform/plot_pyramid.html) from `scikit-image`.
+
+If you submit an image pyramid, napari will automatically detect it as such. To
+turn off automatic detection, you can now pass the `multiscale=True/False`
+parameter to `add_image`. This replaces the `is_pyramid` parameter which has now been removed. In the future, we aim to add multiscale capabilities
+to all our layers.
+
+We have also fixed the bug where too small a tile was shown to file the entire canvas.
+
+### Points with properties
+
+Points are no longer generic coordinates floating in space! Each point can have
+its own personality and character :-). Specifically, each point can have an
+arbitrary number of properties, and attributes such as size, face color, and
+edge color can be determined by those properties. This makes it easier to
+annotate multiple types of points in an image, such as different cell types. To assign properties to points you can pass a dictionary as the `properties` parameter to `add_points`.
+
+### API changes and improvements
+
+We are taking the opportunity of this major release to update a few APIs. We
+hope that the number of users impacted by these changes will be small. In each
+case, we provide an equivalent API for the same functionality.
+
+- `viewer.add_path` has been renamed `viewer.open` and gains the ability to read
+  to any layer type.
+- `add_image(path=...)` and `add_labels(path=...)` have been removed. Users
+  should use `viewer.open(...)` instead with the `layer_type` parameter to
+  force the added data to be a particular layer type.
+- Image pyramids are no longer automatically generated when a dataset is large.
+  This should not affect API compatibility but might affect performance. For
+  most users, this should result in faster startup for large images.
+- `add_image(..., is_pyramid=False)` is now `add_image(..., multiscale=False)`.
+  This will allow us to use a consistent keyword argument when we add
+  multiscale support for other layer types.
+- `layer.to_svg()` has been removed. This functionality is now implemented with
+  `viewer.save('path/to/layer.svg', layer, plugin='svg')` through our plugin architecture.
+
+### And one more thing...
+
+Thanks to the ever-creative Kira Evans, napari will now populate layer names
+based on the name of the variables being visualized:
+
+```python
+import napari
+from skimage import data
+
+camera = data.camera()
+with napari.gui_qt():
+    viewer = napari.view_image(camera)
+    print(viewer.layers[0].name)  # prints "camera"!
+```
```

Comment:
I would also add an example using the walrus operator (Python 3.8+ only).

Reply:
Done!